src/modules/filters/filters.reducer.ts

```diff
@@ -4,7 +4,7 @@ import { FiltersState, FilterType } from './filters.types';
 import * as actions from './filters.actions';
 
 export const INITIAL_STATE: FiltersState = {
-  area: null,
+  area: 'Languages & Frameworks',
```

Comment:
I think we should not set this value as initial one. 
If someone changes the quadrants name in the Contentful, we are not sure that the value exists. 

I recon we should set area as soon as it is fetched from Contentful. I am not sure though how to define which one should be set as initial - the first one? or one of them could be somehow highlighted as initial in Contentful. What you think?